test/features/pageviews/pageviews.js

```diff
@@ -88,6 +92,38 @@ describe('pageviews endpoints', function () {
         });
     });
 
+    it('should not die on bad data in the v column', function () {
+        return preq.post({
+            uri: server.config.globalURL +
+                 fix(insertProjectEndpoint, 'en.wikipedia', '2') +
+                 '/catch'
```

Comment:
I added a second insert endpoint that lets me control if I'm inserting 'views' or 'v'.  I got this test to pass on SQLite, but it doesn't work in Cassandra (I guess because now v is also a bigint, so inserting 'catch' in either of them won't work).  So I'll just remove the test.